README.md

```diff
@@ -33,5 +39,35 @@ Open up [http://localhost:3000](http://localhost:3000) and you should be ready t
 
 If you're used to using the `vercel dev` command provided by [Vercel CLI](https://vercel.com/cli) instead, you can also use that, but it's not needed.
 
-
 <img align="center" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSagMR3FC2rPHvoqZFjHunC5DkmFLdeNzok9d7P8OHW&s" width="50%" alt="Vercel" >
+
+## Contributing
+
+1. If you're a contributor to the repo skip to `Step 2`
+   1. Join the group, check out the [Discord server](https://discord.gg/tTD7PvwpuX)!
+   2. Fork the repo
```

Comment:
Probably we could recommend sharing a user's GitHub profile with Andre so that he can add them to the project.

Reply:
I'd prefer leave this as a conversation for people to have when they join the discord. Maybe someone new takes that responsibility up in the future.